packages/api/src/ontology/Definition.ts

```diff
@@ -118,26 +122,31 @@ export interface ValidPropertyTypes {
   stringTimeseries: any;
 
   attachment: any;
-  geopoint: any;
-  geoshape: any;
+  geopoint: GeoJSON.Point;
+  geoshape: GeoJSON.Geometry;
 }
 
-type MaybeArray<T extends PropertyDefinition, U> = T["multiplicity"] extends
-  true ? Array<U> : U;
+type MaybeArray<T extends { multiplicity?: boolean | undefined }, U> =
+  T["multiplicity"] extends true ? Array<U> : U;
 
 type MaybeNullable<T extends PropertyDefinition, U> = T["nullable"] extends true
   ? U | undefined
   : U;
 
+type Raw<T> = T extends Array<any> ? T[0] : T;
+type Converted<T> = T extends Array<any> ? T[1] : T;
```

Comment:
These feel weird just picking arbitrary indicies out of an array, but I'm assuming they're used to pull things out of a configuration metadata?

Reply:
This was from when i was trying to do something clever and left it in for now.